resources/auth/router.spec.js

```diff
@@ -18,6 +19,12 @@ afterAll(async () => {
   await db.destroy();
 });
 
+// const USER = {
```

Comment:
If this is not a descriptive comment of what your code is doing please remove it.

Reply:
Done.